sceptre/resolvers/project_variables.py

```diff
@@ -24,6 +26,12 @@ def resolve(self):
         :returns: Value at argument location.
         :rtype: str
         """
+        init(autoreset=True)
+        simplefilter('always', DeprecationWarning)
+        warn(Fore.YELLOW + (
+                "The project_variables resolver is being deprecated, and will "
```

Comment:
Try:

```
warn(
    "{0}The project_variables ..."
    "be removed... {1}".format(Fore.YELLOW, Style.RESET_ALL)
)
```

I prefer [string.format()](https://docs.python.org/2/library/string.html#string.Formatter.format) over string concatination with `+`, and we use it more through the rest of the codebase. 

+1 for linting!